### DISCUSSION TITLE
Annotation should be per channel

### DISCUSSION THREAD

**Discussion by mikailweston** [KullmannLab/pyecog2#13] [Ideas]
For multiple channel recordings, ideally we should be able to annotate events that occur in individual channels, or groups of channels. This is the case for 2-8 channel transmitters and neuropixel data

Perhaps using a ROI rectangle that takes into account the underlying channels as well as time. 

A way of sorting, and or filtering the channels would be helpful for annotating only selected channels.

**Comment by mfpleite** [comment]
This capability already exists in the form of assigning different labels for events in different channels. I recognize however that graphically it can get quite cluttered very quickly for large numbers of concurrent channels.
Making the channels a property of the individual annotations would require quite a bit of code refactoring, an easier alternative would be making it a property of the labels, and use it basically just for plotting. Does this sound as a decent compromise?

I have added the filtering and selection of channels to the "to do" list in[ Projects->Pyecog2](https://github.com/KullmannLab/pyecog2/projects/1)
